networkit/cpp/centrality/test/DynBetweennessGTest.cpp

```diff
@@ -87,6 +111,17 @@ TEST_F(DynBetweennessGTest, runDynApproxBetweennessSmallGraphEdgeDeletion) {
     G.addEdge(4, 5);
     G.addEdge(5, 6);
     G.addEdge(5, 7);
+    
+    if (isDirected()) {
+        G.addEdge(4, 1);
+        G.addEdge(3, 0);
+        G.addEdge(5, 2);
+    }
```

Comment:
Since this tiny graph is used in multiple tests, we could implement a function `getTinyGraph` within `DynBetweennessGTest` so that we don't have to replicate this code multiple times.